frontend/src/components/LandingPage.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import  '../css/LandingPage.css';
+
+class LandingPage extends React.Component{
+    //Verifying user login?
+    render(){
+        return(
+            <div className = 'landing-page-container'>
```

Comment:
Make sure to delete the spaces between `classname` the `=` and whatever you are assigning. So in this case it will be 
```suggestion
            <div className='landing-page-container'>
```

Just to keep the code style consistent